runtime/repos.go

```diff
@@ -54,6 +56,23 @@ func (r *Runtime) PutFile(ctx context.Context, instanceID, path string, blob io.
 	return nil
 }
 
+func (r *Runtime) MakeDir(ctx context.Context, instanceID, path string, ignoreIfExists bool) error {
+	repo, release, err := r.Repo(ctx, instanceID)
+	if err != nil {
+		return err
+	}
+	defer release()
+
+	// TODO: Handle ignoreIfExists
```

Comment:
Can we fix this todo (or remove the `ignoreIfExists` option entirely)?

Reply:
Ya i think it makes sense to just remove it. Let it error if the folder exists.